numpy/core/tests/test_simd.py

```diff
@@ -317,11 +317,6 @@ def test_square(self):
         assert square == data_square
         
     def test_max(self):
-        """
-        Test intrinics:
-            npyv_max_##SFX
-            npyv_maxp_##SFX
-        """
```

Comment:
I didn't see these changes during my latest review.

Reply:
Sorry for removing this accidentally
I will fix this in the following PR of maximum and minimum